common-util/src/test/java/org/sunbird/common/models/AppTest.java

```diff
@@ -37,7 +35,7 @@ public static void init() {
   }
 
   @Test
-  public void testGetResourceMethod() throws Exception {
+  public void testGetResourceSuccess() throws Exception {
```

Comment:
testGetResourceSuccess, testPostResourceSuccess, testPostResourceFailureWithWrongUrl and testPatchRequestSuccess should be part of HttpUtilTest.